src/dotnet/Audio.UI.Blazor/Components/AudioRecorder/audio-recorder.ts

```diff
@@ -50,7 +50,9 @@ export class AudioRecorder {
 
     public async canRecord(): Promise<boolean> {
         try {
-            await navigator.mediaDevices.getUserMedia({video: false, audio: true});
+            const stream = await navigator.mediaDevices.getUserMedia({video: false, audio: true});
+            stream.getAudioTracks().forEach(t => t.stop());
```

Comment:
should `canRecord` change state at all?

Reply:
It doesn't - it closes just created stream tracks